_sass/_hero.scss

```diff
@@ -0,0 +1,27 @@
+.hero {
+  align-items: center;
+  background: $ember-orange;
+  display: flex;
+  flex-direction: column;
+  height: 100vh;
+  justify-content: center;
+  width: 100vw;
```

Comment:
Have you gotten bourbon to work with Jekyll? I had a hell of a time with no success for about 20 minutes. I plan to move forward with this and spike on bourbon & Jekyll later.